stock_move_line_qty_picked/models/stock_move.py

```diff
@@ -0,0 +1,52 @@
+# Copyright 2025 Camptocamp SA
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl)
+from collections import defaultdict
+
+from odoo import api, models
+
+
+class StockMove(models.Model):
+    _inherit = "stock.move"
+
+    def _get_picked_quantity(self):
+        if self.picked and any(ml.qty_picked for ml in self.move_line_ids):
+            picked_qty = 0
+            for ml in self.move_line_ids:
+                picked_qty += ml.product_uom_id._compute_quantity(
+                    ml.qty_picked, self.product_uom, round=False
+                )
+            return picked_qty
+        return super()._get_picked_quantity()
+
+    @api.depends("move_line_ids.picked", "move_line_ids.qty_picked")
+    def _compute_quantity(self):
+        picked_moves_ids = []
+        for move in self:
+            if move.picked and any(ml.qty_picked for ml in move.move_line_ids):
+                picked_moves_ids.append(move.id)
+        picked_moves_ids = self.browse(picked_moves_ids)
+        data = self.env["stock.move.line"]._read_group(
+            [("id", "in", picked_moves_ids.move_line_ids.ids)],
+            ["move_id", "product_uom_id"],
+            ["qty_picked:sum"],
+        )
+        sum_qty = defaultdict(float)
```

Comment:
```suggestion
        uom_qties_picked = defaultdict(float)
```
To avoid confusion between qty_sum and sum_qty?

Reply:
Actually most of that code was copied from Odoo (I'll add a comment about it)